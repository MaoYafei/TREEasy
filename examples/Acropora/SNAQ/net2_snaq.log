optimization of topology, BL and inheritance probabilities using:
 hmax = 3,
 tolerance parameters: ftolRel=1.0e-6, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 1.0e-6.
rootname for files: net2_snaq
BEGIN: 10 runs on starting tree (Adif,Aech,(Aten,(Agem,Asub):2.2096797077688):1.0188428797117706);

main seed 99309
seed: 99309 for run 1, 2019-01-13 4:36:17.586

BEGIN SNaQ for run 1, seed 99309 and hmax 3
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 263 (failures: 54) with -loglik=2.33947 and ht_min=[0.11947, 2.75245, 1.74144, 0.00013]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 139		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 37		 37		 6	 |	 0		 0		 --
mvtarget	 27		 27		 0	 |	 0		 0		 --
chdir		 10		 10		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 49		 4		 0	 |	 0		 0		 --
Total		 262		 79		 8	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0
(Adif,(Aten,((Asub,Agem):0.00012667277697588203,#H6:::0.11946971159768806):2.752453832598199):1.7414395671924685,(Aech)#H6:::0.880530288402312);
---------------------
seed: 94247 for run 2, 2019-01-13 4:36:28.038

BEGIN SNaQ for run 2, seed 94247 and hmax 3
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 268 (failures: 54) with -loglik=0.06894 and ht_min=[0.20462, 1.46548, 2.32129, 1.01839]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 142		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 36		 36		 3	 |	 1		 1		 --
mvtarget	 34		 34		 1	 |	 1		 1		 --
chdir		 9		 9		 0	 |	 2		 2		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 46		 0		 0	 |	 0		 0		 --
Total		 267		 80		 5	 |	 4		 4		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.5
(Adif,(Aten,((Agem,Asub):1.0183948198771406,#H6:::0.2046245898536269):1.4654843778544542):2.321287428834008,(Aech)#H6:::0.7953754101463731);
---------------------
seed: 35348 for run 3, 2019-01-13 4:36:33.395

BEGIN SNaQ for run 3, seed 35348 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 251 (failures: 54) with -loglik=0.06894 and ht_min=[0.20463, 1.46548, 2.3213, 1.0184]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 134		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 1	 |	 1		 1		 --
mvtarget	 29		 29		 1	 |	 1		 1		 --
chdir		 12		 12		 1	 |	 2		 2		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 48		 1		 0	 |	 1		 1		 --
Total		 250		 70		 4	 |	 5		 5		 3
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6
((Aten,((Agem,Asub):1.0184018677375133,#H6:::0.20462570840218866):1.4654751817524618):2.3212999959792606,Adif,(Aech)#H6:::0.7953742915978114);
---------------------
seed: 33474 for run 4, 2019-01-13 4:36:39.155

BEGIN SNaQ for run 4, seed 33474 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 249 (failures: 54) with -loglik=0.06894 and ht_min=[0.20463, 2.3213, 1.46547, 1.01841]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 129		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 30		 30		 2	 |	 0		 0		 --
mvtarget	 32		 32		 0	 |	 0		 0		 --
chdir		 11		 11		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 46		 2		 0	 |	 0		 0		 --
Total		 248		 76		 5	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0
(Adif,(((Agem,Asub):1.018406686309984,#H6:::0.2046260753528427):1.4654685012576483,Aten):2.321302724831798,(Aech)#H6:::0.7953739246471573);
---------------------
seed: 68914 for run 5, 2019-01-13 4:36:46.593

BEGIN SNaQ for run 5, seed 68914 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 245 (failures: 54) with -loglik=0.06894 and ht_min=[0.20463, 1.46546, 2.32131, 1.01841]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 131		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 30		 30		 2	 |	 0		 0		 --
mvtarget	 29		 29		 0	 |	 0		 0		 --
chdir		 9		 9		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 45		 2		 2	 |	 0		 0		 --
Total		 244		 72		 6	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0
((Adif,(Aech)#H6:::0.795373569440579):2.3213064090675295,Aten,((Asub,Agem):1.0184100144127244,#H6:::0.20462643055942098):1.4654649689888553);
---------------------
seed: 72711 for run 6, 2019-01-13 4:36:52.192

BEGIN SNaQ for run 6, seed 72711 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 269 (failures: 54) with -loglik=0.06894 and ht_min=[0.20463, 1.46547, 2.3213, 1.01841]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 141		 8		 1	 |	 NA 		 NA 		 NA 
mvorigin	 33		 33		 7	 |	 0		 0		 --
mvtarget	 33		 33		 2	 |	 1		 1		 --
chdir		 9		 9		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 52		 3		 0	 |	 0		 0		 --
Total		 268		 86		 10	 |	 2		 2		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.5
(Aten,(Adif,(Aech)#H6:::0.7953740130222157):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6:::0.2046259869777843):1.465467067093687);
---------------------
seed: 84713 for run 7, 2019-01-13 4:36:59.278

BEGIN SNaQ for run 7, seed 84713 and hmax 3
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 244 (failures: 54) with -loglik=0.06894 and ht_min=[0.20463, 1.0184, 2.32133, 1.46546]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 128		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 31		 31		 4	 |	 0		 0		 --
mvtarget	 31		 31		 2	 |	 0		 0		 --
chdir		 7		 7		 3	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 46		 1		 0	 |	 0		 0		 --
Total		 243		 71		 10	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0
(Adif,(Aech)#H6:::0.7953731307135693,(Aten,((Agem,Asub):1.018404899131462,#H6:::0.20462686928643073):1.4654648053619335):2.3213251152777885);
---------------------
seed: 56576 for run 8, 2019-01-13 4:37:4.64

BEGIN SNaQ for run 8, seed 56576 and hmax 3
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 246 (failures: 54) with -loglik=0.06894 and ht_min=[0.20464, 1.4654, 2.32137, 1.01846]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 129		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 34		 34		 1	 |	 1		 1		 --
mvtarget	 29		 29		 1	 |	 1		 1		 --
chdir		 8		 8		 0	 |	 2		 2		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 45		 0		 0	 |	 0		 0		 --
Total		 245		 72		 3	 |	 4		 4		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.5
(Adif,(Aten,((Asub,Agem):1.0184586795740482,#H6:::0.20463695757925293):1.4653955688633449):2.3213679752061998,(Aech)#H6:::0.7953630424207471);
---------------------
seed: 53158 for run 9, 2019-01-13 4:37:10.565

BEGIN SNaQ for run 9, seed 53158 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 270 (failures: 54) with -loglik=0.06894 and ht_min=[0.20462, 1.46548, 2.32128, 1.01839]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 144		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 35		 35		 5	 |	 3		 3		 --
mvtarget	 31		 31		 1	 |	 3		 3		 --
chdir		 11		 11		 0	 |	 7		 7		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 48		 1		 0	 |	 4		 4		 --
Total		 269		 79		 7	 |	 17		 17		 7
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.4
(Aten,(Adif,(Aech)#H6:::0.7953777808639368):2.3212797151485707,((Agem,Asub):1.01839304469768,#H6:::0.20462221913606315):1.4654755453469355);
---------------------
seed: 57345 for run 10, 2019-01-13 4:37:17.514

BEGIN SNaQ for run 10, seed 57345 and hmax 3 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [122, 25, 25, 4, 0, 42], Nmov [122, 25, 25, 4, 10000, 42]
END optTopLevel: found minimizer topology at step 387 (failures: 54) with -loglik=3.40711 and ht_min=[0.14616, 10.0, 1.86183, 1.61877]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 183		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 50		 50		 16	 |	 0		 0		 --
mvtarget	 54		 54		 1	 |	 0		 0		 --
chdir		 27		 27		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 72		 1		 1	 |	 0		 0		 --
Total		 386		 133		 19	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0
(Aech,(Adif,#H6:::0.1461629256485144):10.0,((Asub,Agem):1.861825873641031,(Aten)#H6:::0.8538370743514856):1.6187712663544507);
---------------------

2019-01-13 4:37:25.839
MaxNet is (Aten,(Adif,(Aech)#H6:::0.7953740130222157):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6:::0.2046259869777843):1.465467067093687); 
with -loglik 0.06893702866723227
