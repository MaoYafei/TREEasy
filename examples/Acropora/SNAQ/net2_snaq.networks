(Aten,(Adif,(Aech)#H6:::0.7953740130222157):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6:::0.2046259869777843):1.465467067093687);, with -loglik 0.06893702866723227 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(Aten,(Adif)#H6:::0.18942535124652354,((Agem,Asub):1.9499844746631656,(Aech,#H6:::0.8105746487534764):2.0145571212291173):2.0145571212291173);, with -loglik 1.6578717366757887
(Aten,(Adif,(Aech,#H6:0.2650889438751081::0.04195954405338236):4.627514108366558):1.2532370984643446,((Agem,Asub):0.0)#H6:4.540703568941324::0.9580404559466177);, with -loglik 7.12327831185079
(#H6:::1.1102230246251565e-16,Adif,(Aech,((Aten)#H6:::0.9999999999999999,(Agem,Asub):1.6537723432967488):1.0308954683387594):2.4999999999999996);, with -loglik 14.781071045577137
