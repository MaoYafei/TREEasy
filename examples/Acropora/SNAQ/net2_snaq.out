(Aten,(Adif,(Aech)#H6:::0.7953740130222157):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6:::0.2046259869777843):1.465467067093687); -Ploglik = 0.06893702866723227
 Dendroscope: (Aten,(Adif,(Aech)#H6):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6):1.465467067093687);
 Elapsed time: 70.350308981 seconds, 10 attempted runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (Aten,(Adif,(Aech)#H6:::0.7953740130222157):2.3213014437652384,((Agem,Asub):1.0184082135032904,#H6:::0.2046259869777843):1.465467067093687);, with -loglik 0.06893702866723227
 (Adif,(((Agem,Asub):1.018406686309984,#H6:::0.2046260753528427):1.4654685012576483,Aten):2.321302724831798,(Aech)#H6:::0.7953739246471573);, with -loglik 0.06893702869184143
 ((Adif,(Aech)#H6:::0.795373569440579):2.3213064090675295,Aten,((Asub,Agem):1.0184100144127244,#H6:::0.20462643055942098):1.4654649689888553);, with -loglik 0.06893702871333454
 ((Aten,((Agem,Asub):1.0184018677375133,#H6:::0.20462570840218866):1.4654751817524618):2.3212999959792606,Adif,(Aech)#H6:::0.7953742915978114);, with -loglik 0.06893702876287836
 (Adif,(Aten,((Agem,Asub):1.0183948198771406,#H6:::0.2046245898536269):1.4654843778544542):2.321287428834008,(Aech)#H6:::0.7953754101463731);, with -loglik 0.06893702885285191
 (Adif,(Aech)#H6:::0.7953731307135693,(Aten,((Agem,Asub):1.018404899131462,#H6:::0.20462686928643073):1.4654648053619335):2.3213251152777885);, with -loglik 0.06893702975988264
 (Aten,(Adif,(Aech)#H6:::0.7953777808639368):2.3212797151485707,((Agem,Asub):1.01839304469768,#H6:::0.20462221913606315):1.4654755453469355);, with -loglik 0.06893703185083871
 (Adif,(Aten,((Asub,Agem):1.0184586795740482,#H6:::0.20463695757925293):1.4653955688633449):2.3213679752061998,(Aech)#H6:::0.7953630424207471);, with -loglik 0.06893704608221513
 (Adif,(Aten,((Asub,Agem):0.00012667277697588203,#H6:::0.11946971159768806):2.752453832598199):1.7414395671924685,(Aech)#H6:::0.880530288402312);, with -loglik 2.3394674041177455
 (Aech,(Adif,#H6:::0.1461629256485144):10.0,((Asub,Agem):1.861825873641031,(Aten)#H6:::0.8538370743514856):1.6187712663544507);, with -loglik 3.4071071785866303
-------
