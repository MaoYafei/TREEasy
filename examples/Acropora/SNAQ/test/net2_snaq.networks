(Adif,(Aten,((Asub,Agem):1.0184104302822294,#H6:::0.20462625086805805):1.4654638985305235):2.321302781677019,(Aech)#H6:::0.7953737491319419);, with -loglik 0.06893702869666156 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(Aten,(Adif)#H6:::0.19043080611131435,((Asub,Agem):1.9524374904548583,(Aech,#H6:::0.8095691938886858):2.008195883986555):2.008195883986555);, with -loglik 1.6582394847930595
(Adif,#H6:::0.15344146894711808,(Aech,((Aten)#H6:::0.8465585310528819,(Asub,Agem):1.8625445304388846):1.6462503396418353):2.7133230768262004);, with -loglik 3.5576987521311594
(Adif,(Aten,((Asub,Agem):0.0)#H6:5.001703688362618::0.9571721846906261):1.2501540372644577,(Aech,#H6:2.3730837554279463::0.04282781530937384):1.6030714373421968);, with -loglik 7.7301057211316335
