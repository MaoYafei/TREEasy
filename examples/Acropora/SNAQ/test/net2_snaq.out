(Adif,(Aten,((Asub,Agem):1.0184104302822294,#H6:::0.20462625086805805):1.4654638985305235):2.321302781677019,(Aech)#H6:::0.7953737491319419); -Ploglik = 0.06893702869666156
 Dendroscope: (Adif,(Aten,((Asub,Agem):1.0184104302822294,#H6):1.4654638985305235):2.321302781677019,(Aech)#H6);
 Elapsed time: 66.571709197 seconds, 10 attempted runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (Adif,(Aten,((Asub,Agem):1.0184104302822294,#H6:::0.20462625086805805):1.4654638985305235):2.321302781677019,(Aech)#H6:::0.7953737491319419);, with -loglik 0.06893702869666156
 (Adif,(Aten,((Asub,Agem):1.018368675693336,#H6:::0.2046203250580613):1.4655042782160468):2.321235924162463,(Aech)#H6:::0.7953796749419387);, with -loglik 0.06893703328397738
 (#H6:::0.20463130872559887,(Asub,Agem):1.0184429689287486,(Aten,(Adif,(Aech)#H6:::0.7953686912744011):2.3213262609217225):1.4654283467280098);, with -loglik 0.06893703360028997
 (Adif,(Aten,((Asub,Agem):1.0182706265107302,#H6:::0.20460828602451933):1.4656599433457662):2.321187095265696,(Aech)#H6:::0.7953917139754807);, with -loglik 0.06893706715040179
 (Aech,((Agem,Asub):1.9553550500740393,(Aten,#H6:::0.18917767591815046):2.008145483680762):2.008145483680762,(Adif)#H6:::0.8108223240818496);, with -loglik 1.6574106518213307
 (((Agem,Asub):1.9553550808138593,(Aten,#H6:::0.18917763703640136):2.0081452883807236):2.0081452883807236,Aech,(Adif)#H6:::0.8108223629635987);, with -loglik 1.657410651821954
 (Aech,((Agem,Asub):1.955355044708557,(Aten,#H6:::0.18917781622798313):2.008145770321525):2.008145770321525,(Adif)#H6:::0.8108221837720168);, with -loglik 1.6574106518296885
 (Aten,((Aech,#H6:::0.81082199656499):2.0081471054123736,(Agem,Asub):1.955356288042793):2.0081471054123736,(Adif)#H6:::0.18917800343501);, with -loglik 1.657410651886345
 ((Aten,((Agem,Asub):4.2968635586790964e-6,#H6:::0.11945927754423433):2.7526176032010587):1.7413734929440956,Adif,(Aech)#H6:::0.8805407224557656);, with -loglik 2.340063841387355
 (Aech,(Adif,(Aten,((Asub,Agem):0.0)#H6:9.967483170891468::0.9557155948233158):1.2656032920345714):10.0,#H6:9.99924901596786::0.04428440517668414);, with -loglik 6.838604478976377
-------
